code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm

```diff
@@ -75,13 +75,20 @@
 	//Parrots will generally sit on their pertch unless something catches their eye.
 	//These vars store their preffered perch and if they dont have one, what they can use as a perch
 	var/obj/parrot_perch = null
-	var/obj/desired_perches = list(/obj/machinery/constructable_frame/computerframe, 		/obj/structure/displaycase, \
-									/obj/structure/filingcabinet,		/obj/machinery/teleport, \
-									/obj/machinery/computer,			/obj/machinery/telecomms, \
-									/obj/machinery/nuclearbomb,			/obj/machinery/particle_accelerator, \
-									/obj/machinery/recharge_station,	/obj/machinery/smartfridge, \
-									/obj/machinery/suit_cycler,	/obj/structure/showcase, \
-									/obj/structure/fountain)
+	var/obj/desired_perches = list(
```

Comment:
Glad it's not just me who has to compulsively tidy these godforsaken lists.

Reply:
It's a curse and a blessing. Mostly a curse.